.github/workflows/test.yml

```diff
@@ -29,7 +29,6 @@ jobs:
       - name: Install npm dependencies
         env:
           CYPRESS_INSTALL_BINARY: '0'
-          PUPPETEER_SKIP_DOWNLOAD: 'true'
```

Comment:
We need to download here now if we want to run the collaboration tests as part of the standard test suite.